Classes/Service/MailService.php

```diff
@@ -54,12 +54,12 @@ public function __construct(BackendUserAuthentication $backendUser = null, Fluid
      * @param  Application $application The application to pull information from
      * @param  string $subject
      * @param  string $body
-     * @param  string $cc
-     * @param  string $bcc
+     * @param  array $cc
+     * @param  array $bcc
      * @param  bool $useBackendUserCredentials Whether the mail should use the current backend user for sender details
      * @return void
      */
-    public function sendReplyMail(Application $application, $subject = "", $body = "", $cc = "", $bcc = "", $useBackendUserCredentials = true)
+    public function sendReplyMail(Application $application, $subject = "", $body = "", $cc = [], $bcc = [], $useBackendUserCredentials = true)
```

Comment:
Are strings still allowed for CC/BCC? If not, maybe add `array` type hint here.

Reply:
I think I'll rather change the doc typehint to array|string to allow both types consistently. Core MailMessage supports both, the service probably should not be the limiting factor here.

Also, forcing arrays will be potentially breaking if someone uses or overrides the service. The method is public...

Reply:
👍